go-controller/pkg/node/node.go

```diff
@@ -539,7 +539,16 @@ func (n *OvnNode) Start(ctx context.Context, wg *sync.WaitGroup) error {
 		go wait.Until(func() {
 			checkForStaleOVSInterfaces(n.name, n.watchFactory.(*factory.WatchFactory))
 		}, time.Minute, n.stopChan)
-		err := n.WatchEndpoints()
+		util.SetARPTimeout()
+		err := n.WatchNamespaces()
+		if err != nil {
+			return fmt.Errorf("failed to watch namespaces: %w", err)
+		}
+		// every 5 minutes cleanup stale conntrack entries if any
```

Comment:
5 minutes seems like too long. How come this value was chosen? ARP should be OK to try more often, maybe every minute?

Reply:
it was a random interval :D I am open to suggestions... 1min is ok for me, TBH I have no idea on a large env what's a good value and what's not. I was thinking let's keep it at 5mins and user decides this is too infrequent we could fix it later, but I didn't want the reverse of too many ARPs.

Reply:
Keeping it at 3mins compromise for now, because the ARP timeout is 50 milliseconds, that means at least we will have time for 60 ARP timeouts within the 3mins, if its 1 min it reduces to 20 ARP timeouts at max worst case scenario.

Reply:
i think this is more reason we need to parallelize the ARPs, we dont want to be waiting that long. Default ARP timeout in linux is 30 seconds. I dont see a problem with running every minute.